packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -252,6 +312,13 @@ export interface JobDefinitionProps {
    * @default - undefined
    */
   readonly timeout?: Duration;
+
+  /**
+   * The platform capabilities required by the job definition.
+   *
+   * @default - undefined
```

Comment:
Defaults should reflect the behavioral default, not the code. From CloudFormation docs:

> If no value is specified, it defaults to EC2

Reply:
done